src/test/java/seedu/address/testutil/TypicalTags.java

```diff
@@ -24,6 +24,7 @@
             .build();
     public static final Tag CS2101 = new TagBuilder().withTagName(VALID_TAG_NAME_CS2101)
             .withFileAddress(VALID_FILE_ADDRESS_CS2101)
+            .withLabels("validLabel")
```

Comment:
If you do not use withLabels, it will still have a label called "testLabel"

Reply:
This conversation is discussed above.